src/main/java/eu/europeana/metis/sandbox/service/record/RecordLogServiceImpl.java

```diff
@@ -24,21 +25,23 @@ public RecordLogServiceImpl(RecordLogRepository repository) {
   }
 
   @Override
-  public void logRecord(Record record) {
-    requireNonNull(record, "Record must not be null");
+  public void logRecordEvent(Event<Record> recordEvent) {
```

Comment:
The log object in this class is misnamed and not used.

Reply:
Is not logging to a log file itself, but what I meant is to log every event in the database, that is where the name logRecordEvent comes from, and that is what I am saving/logging, an event into the "event log". Anyways I could name it saveRecordEvent, but is just a matter of taste from my point of view and I have no strong feelings for it, so let me know what you prefer

Reply:
Sorry I realized what you were talking about, I just removed it